docs/advanced/structure.md

```diff
@@ -248,6 +247,12 @@ def getSampleName() {
 }
 ```
 
+This relies on jsonSlurper which isn't included by default. Import this by adding the following to the top of the file:
+
+```groovy linenums="1"
+import groovy.json.JsonSlurper
+```
```

Comment:
You can also just use JsonSlurper without importing it as `new groovy.json.JsonSlurper()`